registrants/src/UI/embc-registrant/src/app/sharedModules/components/dashboard/dashboard.component.ts

```diff
@@ -70,7 +70,7 @@ export class DashboardComponent implements OnInit {
             essFileData: registrationResult,
             initDialog: false
           },
-          height: '750px',
+          // height: '750px',
```

Comment:
Is this commented out intentionally?

Reply:
yes. After updating Angular, it changed the pop-ups a little bit. By taking out the height, an unnecessary scroller disappears

Reply:
then should this be commented out or deleted?